scripts/gui/thrive_style.css

```diff
@@ -505,21 +712,18 @@ div::-webkit-scrollbar-thumb:hover {
   background-repeat: no-repeat;
 }
 
-
 #OxygenBar {
   background-color: #2dcbf0;
 }
 
 #NitrogenIcon {
-  position: absolute;
-  width: 30px;
-  height: 30px;
-  top: 5px;
-  left: 850px;
-  background-size: 30px 30px;
   background-image: url("../../Textures/gui/bevel/Nitrogen.png");
 }
 
+.TempIcon {
```

Comment:
In my head `TempIcon` is short for `TemporaryIcon` and not for `TemperatureIcon`
This should be renamed `TemperatureIcon` for clarity.